src/herbpy/tsr/generic.py

```diff
@@ -0,0 +1,156 @@
+import numpy
+from prpy.tsr.tsrlibrary import TSRFactory
+from prpy.tsr.tsr import TSR, TSRChain
+
+@TSRFactory('herb', None, 'point')
+def point_obj(robot, transform, manip=None):
+    """
+    @param robot The robot performing the point
+    @param transform The location of where the robot is pointing to
+    @param manip The manipulator to point with. This must be the right arm. 
+    """
+    if manip is None:
+        manip = robot.right_arm
+
+    with manip.GetRobot():
```

Comment:
I would prefer if this created a `RobotStateSaver` that only preserved the active DOFs and active manipulator, instead of using the default options.